src/components/pages/RegisterPage.js

```diff
@@ -7,7 +7,6 @@ class RegisterPage extends Component {
 
     return (
       <div>
-        <h1>Register a new user</h1>

```

Comment:
As our register page and sign in page are quick simple, I'm thinking that we probably don't need to have 'Register Page' and 'Register Form' separately? Same as 'Sign In Page' (currently empty) and 'Sign in Form'? We can discuss this later as it's not important :)